src/components/screens/wiki-documentation-screen.tsx

```diff
@@ -520,60 +520,74 @@ const WikiDocumentationScreen = () => {
     );
   }
 
-  return (
-    <>
-      {formOpen ? (
-        <CreateOrEditArticleForm setFormOpen={setFormOpen} action="create" adminMode={adminMode} />
-      ) : (
-        <>
-          {!adminMode && (
-            <>
-              {renderTitle(strings.wikiDocumentation.cardTitle)}
-              <CarouselArticleCards articles={lastUpdatedArticles} />
-            </>
+return (
+  <>
+    {formOpen ? (
```

Comment:
No need for changes here in this PR but consider going forward we try to need nested ternary/ conditionals in the render to a minumum- ideally no more than 1 nested ternary- and use methods to render conditional components. Just for code clarity. Again this is fine for now, no need to change anything here